nntrainer/tensor/tensor.cpp

```diff
@@ -1283,6 +1284,11 @@ Tensor &Tensor::transpose(const std::string &direction, Tensor &out) const {
 
   out.reshape(dim.transpose(direction));
 
+  if (is_FC) {
+    out.copyData(clone());
```

Comment:
I'm not sure, but why doesn't it copy data from `*this`? It may occur double copy.